adjuster/finagle/src/main/java/zipkin/sparkstreaming/adjuster/finagle/FinagleIssue343Adjuster.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.sparkstreaming.adjuster.finagle;
+
+import com.google.auto.value.AutoValue;
+import zipkin.Annotation;
+import zipkin.BinaryAnnotation;
+import zipkin.Span;
+import zipkin.internal.Util;
+import zipkin.sparkstreaming.Adjuster;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains adjustments for finagle issue 343 (https://github.com/twitter/finagle/issues/343).
+ * If applied, this adjuster drops finagle.flush annotation.
+ */
+@AutoValue
+public abstract class FinagleIssue343Adjuster extends Adjuster{
```

Comment:
If it were me, I'd make this public final and then with a package-protected constructor. Add to that a factory method like `public static FinagleIssue343Adjuster create()` which calls the package protected constructor.